src/libs/validationHandler.ts

```diff
@@ -0,0 +1,88 @@
+import { NextFunction, Request, Response } from 'express';
+export default ( config ) => ( req: Request, res: Response, next: NextFunction  ) => {
+    const errors = [];
+    console.log( 'Inside ValidationHandler Middleware' );
+    console.log( req.body );
+    console.log( req.query );
+    console.log(Object.keys( req.query ).length );
+    const keys = Object.keys( config );
+    keys.forEach((key) => {
+      const obj = config[key];
+      console.log('key is' , key);
+      const values = obj.in.map( ( val ) => {
+          return req[ val ][ key ];
+      });
```

Comment:
`values` will return an array but you only checking for value at index 0 what if we have 2 locations inside in key.
For example skip and limit can be a part of query string or body.